### PR TITLE
Seed directory for WebRTC Statistic spec

### PR DESCRIPTION
**Pull request by dontcallmedom**
https://w3c.github.io/webrtc-stats/webrtc-stats.html

### CODE REVIEW

**Review by fippo — APPROVED** [review]

File: webrtc-stats/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The following 4 test cases in the `webrtc/` directory test some of the mandatory-to-implement stats defined in WebRTC Statistics:
+
+* `webrtc/getstats.html`
```
Comment (foolip) [inline webrtc-stats/README.md]: These paths have changed, right?

**Review by foolip — DISMISSED** [review]
Looks good with paths double checked.

File: webrtc-stats/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+* `webrtc/RTCPeerConnection-getStats.https.html`
+* `RTCPeerConnection-track-stats.https.html`
+* `RTCRtpReceiver-getStats.https.html`
+* `RTCRtpSender-getStats.https.html`
```
Comment (henbos) [inline webrtc-stats/README.md]: Why doesn't all paths start with webrtc/...?

**Comment by foolip** [comment]
See skepticism in https://github.com/web-platform-tests/wpt/pull/10610#pullrequestreview-124306787.

File: webrtc-stats/README.md (hunk 2 above)
Comment (alvestrand) [reply webrtc-stats/README.md]: They should, but it doesn't matter much - it's a README.

File: webrtc-stats/README.md (hunk 1 above)
Comment (alvestrand) [reply webrtc-stats/README.md]: Not those.

**Review by alvestrand — APPROVED** [review]
Let's get the show started.

**Comment by foolip** [comment]
@dontcallmedom can you also add a META.yaml with suggested reviewers?

**Comment by zcorpan** [comment]
@foolip I updated #10610 to have the META.yml. @dontcallmedom could cherry-pick into this branch, or we can land both PRs.

**Comment by alvestrand** [comment]
Both PRs landed.

**Pull request merged**